src/cdk/schematics/ng-add/index.spec.ts

```diff
@@ -4,6 +4,10 @@ import {COLLECTION_PATH} from '../index.spec';
 import {createTestApp, getFileContent} from '../testing';
 import {addPackageToPackageJson} from './package-config';
 
+interface PackageJson {
+  dependencies: {[key: string]: string};
```

Comment:
Can be simplified to `Record<string, string>`. Also this is being repeated in a few test files, should we reuse it?

Reply:
I am not sure the best option for a location for us to put this common interface. Any ideas?

Reply:
I don't see a good place to add it. Let's just leave it as it is for now.